src/components/ui/Button3.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+
+const Button3 = ({answers, checked, id , sent, selected, correct , className, clicked}) => {
+    const [sel, setSel] = useState(null)
+    let styles
+    
+    function handleClick(answer) {
+        setSel(answer)
+        clicked(id,answer)
+    }
+    return ( 
+        <div href="/" className="flex  gap-10" >
+            {answers.map((item, index)=>{
+                
+                if(checked){
+                    styles={ backgroundColor: sel === item && '#D6DBF5' }
+                    if(sent){
+                        if(sel === item){
+                            if(correct === item){
+                                styles={ backgroundColor: '#94D7A2' }
+                            }
+                            else{
+                                styles={ backgroundColor: '#F8BCBC' }
+                            }
+                            
+                        }
+                        if(item === correct){
+                            styles={ backgroundColor: '#94D7A2' }
+                        }
+                    }
+                    
+                }
```

Comment:
I think it should be much ideal to use a switch case instead of using an else-if statement, as the switch is much faster and clear. but if you feel like it's okay with you to implement using else-if also it can be good as this codebase is not really big.